.github/workflows/openwrt-build.yml

```diff
@@ -28,16 +28,22 @@ jobs:
           cd openwrt
           TARGET=${{ matrix.target }} make templates build-img package 2>&1 | tee ${{ matrix.target }}-build.log
       - name: 'Upload openwrt build artifact tarball'
-        uses: actions/upload-artifact@v3.1.1
+        uses: actions/upload-artifact@v4.6.0
         with:
           name: ${{ matrix.target }}-openwrt-build-artifacts
           # Cant use relative pathing .. or . for artifacts action
           # Also dont bother with '${{ github.workspace }}'
           path: |
             openwrt/build/artifacts/*.tar.gz
           retention-days: 30
+          # Since we cant turn off compression set to lowest level
+          compression-level: 0
+          # rerunning jobs overwrites artifacts from previous runs
+          # this was the default behavior in v3.x
+          # ref: https://github.com/actions/upload-artifact/blob/v3.1.1/README.md?plain=1#L152
+          overwrite: true
       - name: 'Upload openwrt build log'
-        uses: actions/upload-artifact@v3.1.1
+        uses: actions/upload-artifact@v4.6.0
```

Comment:
It's _probably_ never going to happen but do we want to use `if-no-files-found:` just in case?

Reply:
Added this to both jobs